src/py/flwr/simulation/ray_transport/ray_client_proxy.py

```diff
@@ -44,16 +44,22 @@ class RayActorClientProxy(ClientProxy):
     """Flower client proxy which delegates work using Ray."""
 
     def __init__(
-        self, client_fn: ClientFn, cid: str, actor_pool: VirtualClientEngineActorPool
+        self,
+        client_fn: ClientFn,
+        partition_id: int,
+        node_id: int,
```

Comment:
nit: let's keep the order the same everywhere
```suggestion
        node_id: int,
        partition_id: int,
```